src/event.cc

```diff
@@ -204,7 +204,7 @@ EventScheduler::timeRemaining() const
     if (tasks->when <= current_dtime) // we are on time or late
         return 0; // fire the event ASAP
 
-    const double diff = tasks->when - current_dtime; // microseconds
+    const double diff = tasks->when - current_dtime; // seconds
```

Comment:
I did not change the type of `diff` variable to `auto` because this code secretly relies on `diff` type being an arithmetic type. For example, when using std::chrono::duration for `diff`, `1000*diff` expression below will compile but will not convert seconds to milliseconds as intended. Event::when type, this whole method (including its return type), and method callers should be reworked using std::chrono, but this is a minimal PR fixing (my 2013 commit aa14d759) bug that wasted 30 minutes of my time.

Said that, if you insist on `auto` upgrade, just apply this suggestion:

```suggestion
    const auto diff = tasks->when - current_dtime; // seconds
```